package.json

```diff
@@ -53,31 +53,32 @@
   "homepage": "https://github.com/alex3165/react-mapbox-gl#readme",
   "dependencies": {
     "deep-equal": "^1.0.1",
-    "mapbox-gl": "^0.32.1",
     "reduce-object": "^0.1.3",
-    "supercluster": "^2.2.0"
+    "supercluster": "^2.3.0"
   },
   "peerDependencies": {
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1"
+    "mapbox-gl": "^0.34.0",
```

Comment:
Nice I didn't know they changed the `package.json` to target the bundle of mapbox-gl instead of the source files 👍